### DISCUSSION TITLE
Error when running

### DISCUSSION THREAD

**Discussion by rxmxr** [jdholtz/auto-southwest-check-in#294] [Q&A] [answered]
Hi--This is a spectacular project--thank you! I'm pretty green with Python but am hoping you can point me in the right direction.

I created a config.json file and get the following error as the process is running. Also get this same error if attempting to check-in for flights in an entire account (using python southwest.py [user] [pass]. 

Any thoughts as to what might be going on?

2024-08-28 12:44:42 DEBUG Process-4[webdriver:149]: Loading Southwest home page (this may take a moment)
Process Process-4:
Traceback (most recent call last):
  File "C:\Python312\Lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Python312\Lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\XXUser\auto-southwest-check-in\lib\reservation_monitor.py", line 44, in monitor
    self._monitor()
  File "C:\Users\XXUser\auto-southwest-check-in\lib\reservation_monitor.py", line 62, in _monitor
    should_exit = self._check()
                  ^^^^^^^^^^^^^
  File "C:\Users\XXUser\auto-southwest-check-in\lib\reservation_monitor.py", line 83, in _check
    self.checkin_scheduler.refresh_headers()
  File "C:\Users\XXUser\auto-southwest-check-in\lib\checkin_scheduler.py", line 50, in refresh_headers
    webdriver.set_headers()
  File "C:\Users\XXUser\auto-southwest-check-in\lib\webdriver.py", line 84, in set_headers
    driver = self._get_driver()
             ^^^^^^^^^^^^^^^^^^
  File "C:\Users\XXUser\auto-southwest-check-in\lib\webdriver.py", line 152, in _get_driver
    driver.click("(//div[@data-qa='placement-link'])[2]")
  File "C:\Python312\Lib\site-packages\seleniumbase\core\sb_driver.py", line 59, in click
    page_actions.click(self.driver, *args, **kwargs)
  File "C:\Python312\Lib\site-packages\seleniumbase\fixtures\page_actions.py", line 1534, in click
    element.click()
  File "C:\Python312\Lib\site-packages\selenium\webdriver\remote\webelement.py", line 94, in click
    self._execute(Command.CLICK_ELEMENT)
  File "C:\Python312\Lib\site-packages\selenium\webdriver\remote\webelement.py", line 395, in _execute
    return self._parent.execute(command, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "C:\Python312\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <div data-qa="placement-link">...</div> is not clickable at point (195, 220). Other element would receive the click: <h3 class="popup-title">...</h3>
  (Session info: chrome=127.0.6533.122)
Stacktrace:
        GetHandleVerifier [0x00007FF65CD59642+30946]
        (No symbol) [0x00007FF65CD0E3D9]
        (No symbol) [0x00007FF65CC06FDA]
        (No symbol) [0x00007FF65CC5FEEE]
        (No symbol) [0x00007FF65CC5D962]
        (No symbol) [0x00007FF65CC5AE7B]
        (No symbol) [0x00007FF65CC5A095]
        (No symbol) [0x00007FF65CC4C271]
        (No symbol) [0x00007FF65CC7CA6A]
        (No symbol) [0x00007FF65CC4BBB6]
        (No symbol) [0x00007FF65CC7CC80]
        (No symbol) [0x00007FF65CC9B041]
        (No symbol) [0x00007FF65CC7C813]
        (No symbol) [0x00007FF65CC4A6E5]
        (No symbol) [0x00007FF65CC4B021]
        GetHandleVerifier [0x00007FF65CE8F84D+1301229]
        GetHandleVerifier [0x00007FF65CE9BDC7+1351783]
        GetHandleVerifier [0x00007FF65CE92A13+1313971]
        GetHandleVerifier [0x00007FF65CD8DD16+245686]
        (No symbol) [0x00007FF65CD1759F]
        (No symbol) [0x00007FF65CD13814]
        (No symbol) [0x00007FF65CD139A2]
        (No symbol) [0x00007FF65CD0A3FF]
        BaseThreadInitThunk [0x00007FF931397374+20]
        RtlUserThreadStart [0x00007FF9327DCC91+33]

**Comment by dmytrokoren** [comment]
This means that "Something went wrong message" popup when you load southwest page. This is due that application detects you as a bot. Please run develop branch instead of master.

**Comment by rxmxr** [comment]
Thanks very much; running develop and still getting the same error. Is this likely to clear up if I wait a day or two?

**Reply by dmytrokoren** [reply]
> Thanks very much; running develop and still getting the same error. Is this likely to clear up if I wait a day or two?

No problem, are you running in docker? What's the os - Windows, Mac, Linux?

**Reply by rxmxr** [reply]
Windows; not running Docker.

**Reply by dmytrokoren** [reply]
> Windows; not running Docker.

Okay I'll have to check and let you know

**Comment by dmytrokoren** [comment]
> Thanks very much; running develop and still getting the same error. Is this likely to clear up if I wait a day or two?

It's working fine for me in windows. Make sure you checkout develop branch before running. I have no problem executing on my end.

**Reply by rxmxr** [reply]
I downloaded the develop branch and it executes just fine in some cases. For example, if I have 5 reservations to check (without logging in), it will check the first 2 or 3 successfully, and then give me the error I posted above for the 4th and 5th.

Similarly, if I check by account, I will now get the error immediately (or "Notice: Encountered a Too Many Requests error while logging in for xxxxxx. Skipping reservation retrieval until next interval") for that account.

I truly appreciate the help!

**Comment by dmytrokoren** [comment]
You'll find more answers here:
https://github.com/jdholtz/auto-southwest-check-in/issues/277

**Comment by dc-IV-AW** [comment]
My flight check-in is about an hour from typing this comment, but in my case I was able to get beyond the error by tethering from my T-Mobile based phone. When using Spectrum Home Internet, it almost seems like an IP ban at the Airbnb, and I could never get past too many logins, even after waiting several hours while running [Latest](https://github.com/jdholtz/auto-southwest-check-in/releases/tag/v8.0).

Edit to add that even when tethering, if I did not change my IP on my cellphone by putting on Airplane Mode for a short time, I was getting the too many attempts error again. Once I changed my IP, I was able to run two sessions, and checked in on two different SW accounts, and got good line positions.

**Comment by daddyodeals** [comment]
I fixed mine by commenting out this line in webdriver.py.  Everything seems to be working fine for the past couple of weeks since I made the change.  

#driver.click("(//div[@Data-QA='placement-link'])[2]")

**Comment by jdholtz** [comment, marked as answer]
With the recent changes to the webdriver, this should be fixed now.
